pkg/sql/backfill/backfill.go

```diff
@@ -214,7 +228,14 @@ func (cb *ColumnBackfiller) InitForDistributedUse(
 	// entire backfill process.
 	flowCtx.TypeResolverFactory.Descriptors.ReleaseAll(ctx)
 
-	return cb.init(evalCtx, defaultExprs, computedExprs, desc)
+	return cb.init(evalCtx, defaultExprs, computedExprs, desc, mon)
+}
+
+// Close frees the resources used by the ColumnBackfiller.
+func (cb *ColumnBackfiller) Close(ctx context.Context) {
+	cb.boundAccount.Close(ctx)
```

Comment:
I think that these calls will NPE if they haven't been initialized (like in the case you pass in a nil `mon`.  Should you check for `cb.mon == nil` before closing these things? What situations would you get a nil mon anyway, is it just tests?